src/components/views/messages/TextualBody.tsx

```diff
@@ -238,18 +238,19 @@ export default class TextualBody extends React.Component<IBodyProps, IState> {
     }
 
     private highlightCode(code: HTMLElement): void {
-        if (code.textContent.length > MAX_HIGHLIGHT_LENGTH) {
+        const codeTextContent = code.textContent ?? "";
+        if (codeTextContent.length > MAX_HIGHLIGHT_LENGTH) {
             console.log(
                 "Code block is bigger than highlight limit (" +
-                    code.textContent.length +
+                    codeTextContent.length +
                     " > " +
                     MAX_HIGHLIGHT_LENGTH +
                     "): not highlighting",
             );
             return;
         }
 
-        let advertisedLang;
+        let advertisedLang = "";
```

Comment:
I'm not sure I understand why this change is here: we're using it as a condition on the 'if' block so it being undefined or a string feels more correct?